src/test/integration/util/create.ts

```diff
@@ -0,0 +1,140 @@
+import { WebDriver, Workbench, InputBox, BottomBarPanel } from 'vscode-extension-tester';
+import { DialogHandler } from 'vscode-extension-tester-native';
```

Comment:
<!--
  0 failure: 
  1 warning:  src/test/integrat...
  
  
  DangerID: danger-id-Danger;
  File: src/test/integration/util/create.ts;
  Line: 2;
-->

- :warning: src/test/integration/util/create.ts line 2 – 'DialogHandler' is defined but never used. (@typescript-eslint/no-unused-vars)